realm/realm-library/src/main/cpp/io_realm_internal_LinkView.cpp

```diff
@@ -19,10 +19,13 @@
 
 using namespace realm;
 
+static void finalize_link_view(jlong ptr);
+
 JNIEXPORT void JNICALL Java_io_realm_internal_LinkView_nativeClose
```

Comment:
@nhachicha This is only called by the bulk insert. I think we could remove the calling from there and just rely on GC to free it?